pkg/policy/selectorcache.go

```diff
@@ -175,6 +175,9 @@ func newIdentity(nid identity.NumericIdentity, lbls labels.LabelArray) scIdentit
 }
 
 func getIdentityCache(ids cache.IdentityCache) scIdentityCache {
+	if ids == nil {
+		return make(map[identity.NumericIdentity]scIdentity)
+	}
```

Comment:
This does not seem necessary as len of a nil map is 0 and range over it is a no-op?

Reply:
I want to make sure that the `idCache` field for a `SelectorCache` is always non-nil since we do lookups into it directly in the `SelectorCache`.

Reply:
How does this change help with that? If `ids` is `nil`, line 183 won't executed.

Reply:
You're right, this isn't necessary. I erroneously thought that `len(nil)` would panic, and I was wrong.